esmvalcore/preprocessor/_regrid.py

```diff
@@ -306,6 +307,61 @@ def _attempt_irregular_regridding(cube, scheme):
     return False
 
 
+def extract_location(cube, location, scheme):
+    """Extract a point using a location name, with interpolation.
+
+    Extracts a single location point from a cube, according
+    to the interpolation scheme ``scheme``.
+
+    The function just retrieves the coordinates of the location and then calls
+    the ``extract_point`` preprocessor.
+
+    It can be use to locate cities and villages, but also mountains or other
+    geographical locations.
+
+    Parameters
+    ----------
+    cube : cube
+        The source cube to extract a point from.
+
+    location : str
+        The reference location. Examples: 'mount everest',
+        'romania','new york, usa'
+
+    scheme : str
+        The interpolation scheme. 'linear' or 'nearest'. No default.
+
+    Returns
+    -------
+    Returns a cube with the extracted point, and with adjusted
+    latitude and longitude coordinates.
+
+    Raises
+    ------
+    ValueError:
+        If location is not supplied as a preprocessor parameter.
+    ValueError:
+        If scheme is not supplied as a preprocessor parameter.
+    ValueError:
+        If given location cannot be found by the geolocator.
+    """
+    if location is None:
+        raise ValueError("Location needs to be specified."
+                         " Examples: 'mount everest', 'romania',"
+                         " 'new york, usa'")
+    if scheme is None:
+        raise ValueError("Interpolation scheme needs to be specified."
+                         " Use either 'linear' or 'nearest'.")
+    geolocator = Nominatim(user_agent='esmvalcore')
+    geolocation = geolocator.geocode(location)
+    if geolocation is None:
+        raise ValueError(f'Requested location {location} can not be found.')
+    logger.debug("Extracting data for %s (%s ºN, %s ºW)", geolocation,
```

Comment:
```suggestion
    logger.debug("Extracting data for %s (%s °N, %s °E)", geolocation,
```

Sorry, my latest edits weren't included, this needs to be °E, not °W

Reply:
no idea why E now W but I'll trust you, I don't like East though :laughing:

Reply:
Well for a place in Germany I got `50N 8W`, and Germany is clearly east of London :grimacing:

Reply:
as it stands, it's way more West in terms of mentality :rofl: